torch/onnx/symbolic.py

```diff
@@ -254,6 +254,11 @@ def addmm(g, self, mat1, mat2, beta, alpha):
     return g.op("Gemm", mat1, mat2, self, beta_f=_scalar(beta), alpha_f=_scalar(alpha))
 
 
+def linear(g, input, weight, bias):
+    # TODO: How does defined() behave under tracing?
```

Comment:
Gemm in onnx only accepts 2d tensor. It seems we may have 3d or 4d tensors in linear. Shall we put some check here? If it is not 2d, add some reshape?